.travis.yml

```diff
@@ -1,8 +1,13 @@
 language: objective-c
-rvm: 2.2.3
+cache:
+  - bundler
+  - cocoapods
+rvm: 2.3.1
 install:
     - bundle install
-    - bundle exec pod install
+    - bundle exec pod repo remove master
```

Comment:
I finally manage to get the CI back to business again, please review only the latest changes